ansible/roles/cyhy_feeds/tasks/main.yml

```diff
@@ -0,0 +1,221 @@
+    object: gpg_keys.trust
+    dest: /tmp/gpg_keys.trust
+    mode: get
+  become: no
+
+=======
+>>>>>>> 9eef88aa0b349b80d16b734abe790d3a450606af
+#
+# Copy extract config to new instance
+#
+- name: Copy config file for moe data extractor
+  copy:
+    src: /tmp/cyhy-data-extract.cfg
+    dest: /var/cyhy/scripts/cyhy-feeds/cyhy-data-extract.cfg
+    mode: 0644
+
+- name: Copy gpg public key for moe data extract
+  copy:
+    src: /tmp/ncats_lab_public_gpg.key
+    dest: /var/cyhy/scripts/cyhy-feeds/ncats_lab_public_gpg.key
+    mode: 0644
+
+<<<<<<< HEAD
+- name: Copy john's public key for moe data extract
+  copy:
+    src: /tmp/john_public.key
+    dest: /var/cyhy/scripts/cyhy-feeds/john_public.key
+    mode: 0644
+
+- name: Copy gpg private key for moe data extract
+  copy:
+    src: /tmp/ncats_lab_private_gpg.key
+    dest: /var/cyhy/scripts/cyhy-feeds/ncats_lab_private_gpg.key
+    mode: 0644
+
+- name: Copy gpg trust file
+  copy:
+    src: /tmp/gpg_keys.trust
+    dest: /var/cyhy/scripts/cyhy-feeds/gpg_keys.trust
+    mode: 0644
+
+=======
+>>>>>>> 9eef88aa0b349b80d16b734abe790d3a450606af
+- name: Create the /var/cyhy/scripts/cyhy-feeds/cyhy_extracts directory
+  file:
+    path: /var/cyhy/scripts/cyhy-feeds/cyhy_extracts
+    state: directory
+
+<<<<<<< HEAD
+- name: import lab public key
+  shell: "gpg2 --trustdb-name /home/kyle.evers/.gnupg/trustdb.gpg --import /var/cyhy/scripts/cyhy-feeds/ncats_lab_public_gpg.key"
+  
+- name: import john public key
+  shell: "gpg2 --trustdb-name /home/kyle.evers/.gnupg/trustdb.gpg --import /var/cyhy/scripts/cyhy-feeds/john_public.key"
+
+- name: import lab private key
+  shell: "gpg2 --allow-secret-key-import --import /var/cyhy/scripts/cyhy-feeds/ncats_lab_private_gpg.key"
+
+- name: Add trust
+  shell: "gpg2 --import-ownertrust /var/cyhy/scripts/cyhy-feeds/gpg_keys.trust"
+
+- name: Delete johns public key
+  local_action:
+    module: file
+    path: /tmp/john_public.key
+    state: absent
+  become: no
+=======
+- name: Add ncats public key to gpg key ring
+  shell: gpg2 --allow-non-selfsigned-uid --import /var/cyhy/scripts/cyhy-feeds/ncats_lab_public_gpg.key
+>>>>>>> 9eef88aa0b349b80d16b734abe790d3a450606af
+
+- name: Delete local copy config file for moe data extractor
+  local_action:
+    module: file
+    path: /tmp/cyhy-data-extract.cfg
+    state: absent
+  become: no
+
+- name: Delete local copy gpg public key for moe data extract
+  local_action:
+    module: file
+    path: /tmp/ncats_lab_public_gpg.key
+    state: absent
+  become: no
+
+<<<<<<< HEAD
+- name: Delete local copy gpg public key for moe data extract
+  local_action:
+    module: file
+    path: /tmp/ncats_lab_private_gpg.key
+    state: absent
+  become: no
+
+- name: Delete gpg trust file
+  local_action:
+    module: file
+    path: /tmp/gpg_keys.trust
+    state: absent
+  become: no
+
+=======
+>>>>>>> 9eef88aa0b349b80d16b734abe790d3a450606af
+- name: Create /etc/cyhy/cyhy.conf with commander credentials
```

Comment:
Just a reminder that this step will have to be deconflicted (possibly removed) once we move cyhy-feeds on to the Production mongo instance.

Reply:
Made this an issue to not lose track: https://github.com/dhs-ncats/cyhy_amis/issues/129#issue-372051958